dockerfiles/ci/centos/6/php-5.5/configure.sh

```diff
@@ -11,13 +16,16 @@
     --enable-sockets \
     --enable-zip \
     --with-curl \
+    --with-fpm-user=www-data \
+    --with-fpm-group=www-data \
+    --with-libedit \
     --with-mhash \
     --with-mysqli=mysqlnd \
     --with-openssl \
     --with-pdo-mysql=mysqlnd \
-    --with-pdo-pgsql \
-    --with-pdo-sqlite \
     --with-pear \
     --with-readline \
     --with-zlib \
+    --without-pdo-sqlite \
+    --without-sqlite3 \
```

Comment:
Why the change here?

Reply:
Very good question. IIRC this was disabled in the PHP 8 container because [PHP 7.4+ requires sqlite3 v3.7.4](https://www.php.net/manual/en/sqlite3.requirements.php) which would have required building it from source. But I don't think there's a reason to disable it on PHP 7.3 and below so I'll add this back to all of the PHP versions that support the old version of sqlite3.